.env.http.example

```diff
@@ -43,6 +43,7 @@ OPERATOR_KEY_MAIN=               # Operator private key used to sign transaction
 # FEE_HISTORY_MAX_RESULTS=10     # Maximum number of results to return for eth_feeHistory
 # TX_DEFAULT_GAS=400000          # Default gas for transactions that don't specify gas
 # MAX_TRANSACTION_FEE_THRESHOLD=15000000 # Max transaction fee paid by relay operator account
+# MAX_GAS_ALLOWANCE_HBAR=0       # The maximum amount, in hbars, that the relay is willing to pay to complete the transaction in case the senders don't provide enough funds.
```

Comment:
See comment in `docs/configuration.md` about notation.

Reply:
Hi @acuarica , the amount is actually in hbar. I noticed the js doc was wrong while working on this PR, so I fixed it (https://github.com/hiero-ledger/hiero-sdk-js/commit/b666e2ad37c2b38178ead46bf8e4de4d18b2fb28) :D